src/lib/djerba/plugins/wgts/snv_indel/R/process_snv_data.r

```diff
@@ -71,15 +73,19 @@ if (is.null(cnfile)) {
   cn <- fread(cnfile)
   purity <- as.numeric(purity)
 
-  # Merge dataframes to get only that information needed for the LOH calculation
-  # Note: this does not throw an error if df_filt_oncokb is empty (i.e. only contains a header).
-  calc_df = merge(df_filt_oncokb[,c("Hugo_Symbol", "tumour_vaf")], cn, by="Hugo_Symbol")
+  # Merge dataframes to get only that information needed for the LOH calculation only if there are SNVs.
+  if (calculate_LOH == FALSE ) {
```

Comment:
Would be simpler and more usual to flip the order:
if CONDITION do ACTION else do OTHER ACTION

instead of:
if CONDITION==FALSE do OTHER ACTION else do ACTION